app/person.rb

```diff
@@ -9,13 +9,13 @@ def initialize(age, name = 'Unknown', parent_permission: true)
     @parent_permission = parent_permission
   end
 
-  def can_use_services?(age)
-    of_age(age) || @parent_permission
+  def can_use_services?
+    of_age || @parent_permission
   end
```

Comment:
### As i given code in the previous review of mine. You missed some points in that code 🤔

**First Point**

- [ ] You are using of_age method but you forgot that there is a question mark `?` at the end of this method name 😨 Kindly use this method as `of_age?()` instead of `of_age` 😊
- [ ] And you need to call that function using callbacks means with parantheses such as `of_age()` as you can see in the above change 👍

Reply:
Now, I am getting really confused! Running rubocop removes the "()" . I f I put it there my linters are failing. If I submit it will be marked as "Invalid code review request". What should I do now?

Reply:
@j-tee, the functions in ruby can be called without parathenses. Just ensure that the name of the function is similar the one you defined.